.circleci/config.yml

```
@@ -4,30 +4,79 @@ workflows:
   version: 2
   pushbuild:
     jobs:
-      - debian_build
+      - debian_build:
+          name: "amd_debian_build"
+          resource_class: 'small'
+          platform: amd64
+
+      - debian_build:
+          name: "arm_debian_build"
```

```suggestion
          name: "arm64_debian_build"
```
It might be worth specifying, just in case we transition to wanting to support arm32 again in the future.